profiles/creality/README.md

```diff
@@ -0,0 +1,21 @@
+# Creality CR10-s Profile
```

Comment:
We should remove this. Instead, points out the relevant settings for stacks in the stacking readme (see above).